src/Commands/EnvCreateCommand.php

```diff
@@ -150,18 +150,24 @@ public function createBuildEnv(
             $site_name = $site->getName();
             $project = $this->projectFromRemoteUrl($metadata['url']);
             $dashboard_url = "https://dashboard.pantheon.io/sites/{$site_id}#{$multidev}";
+            $pr_id = '';
+            if (preg_match('#^pr-([\d]+)$#', $multidev, $matches)) {
+                $pr_id = $matches[1];
+            }
```

Comment:
@greg-1-anderson @ataylorme The one thing I don't love about this PR is pulling the `$pr_id` from `$multidev`.

The existing implementation is a little bit more robust because the commit `sha` is the same between Pantheon and the Git provider. We don't have same level of precision for the pull request ID, however.

Open to your suggestions on how to best derive this value. Another option would be to read the `PR_NUMBER` environment variable directly, but I feel like that's even more magical.

Reply:
@danielbachhuber I think this is okay since `$doNotify` is only true when a new multidev is created. The multidev environment is `pr-` by default, though users can configure it to be the `ci-` variant.

Go ahead and fetch the PR number from the multidev name, as you are doing now, as well as from the environment variable and only commented on the pull request if they match.

Reply:
> Go ahead and fetch the PR number from the multidev name, as you are doing now, as well as from the environment variable and only commented on the pull request if they match.

Which environment variable? It seems like there are at least a couple and I'm not sure which we need to respect.

Reply:
Either `PR_NUMBER` or `TERMINUS_ENV` would work but go with `TERMINUS_ENV` because it is required, where users may not have `PR_NUMBER` set. `TERMINUS_ENVV` will have the `pr-` prefix so you will need to parse the ID from that, just like you are doing with `$multidev`

Reply:
> `TERMINUS_ENV` because it is required,

Why is `TERMINUS_ENV` required? I didn't have it set locally when it was testing.

On second thought, I don't think we need to inspect an environment variable too. Unless @greg-1-anderson has some alternative idea, I'm happy with the current implementation.

Reply:
>  I don't think we need to inspect an environment variable too

I'm okay with that.